api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -246,7 +246,9 @@ def get(self, key: str, default: Any | None = None) -> Any:
             ):  # The only offers without oa neither in themselves nor in venues are the numerics ones.
                 return None
             offererAddress = GetOffererAddressWithIsEditableResponseModel.from_orm(offerer_address)
-            offererAddress.label = offererAddress.label or self._obj.venue.common_name
+            offererAddress.label = (
+                self._obj.venue.common_name if not offererAddress.isEditable else offererAddress.label
```

Comment:
Petite suggestion : 

```suggestion
                offererAddress.label if offererAddress.isEditable else self._obj.venue.common_name
```

(un simple if est plus facile à lire qu'un `if not`)